scripts/app-build-deploy.sh

```diff
@@ -16,21 +16,13 @@ echo "Service Name Workspace: ${SERVICE_NAME_WORKPACE}"
 
 cd ./application/${SERVICE_NAME}
 
-rsync -Rr ./ ./chalicebuildtmp
-cd ./chalicebuildtmp
-cp -r ../../../application-utils/* ./chalicelib/
-pip install -r requirements.txt --target .
+zip -r deployment.zip .
```

Comment:
Above we have identical copies of the utitlities.py per lambda - which will trip our code duplication quality gate. The true source is held once here - application-utils/common/utilities.py and copied into the relevant lambda folder (both here as part of deployment and as part of unit testing). I think we need to keep the line that copies the application-utils folder before we zip up the deployment package